### ISSUE TITLE
NavigationProgress component didn't pass aria-label to Progress component

### ISSUE THREAD

**Issue by zulianrizki** [mantinedev/mantine#2517]
### What package has an issue

@mantine/nprogress

### Describe the bug

Currently, there are no easy way to add `aria-label` to `NavigationProgress`. From the source code, it doesn't have any `aria-label` props although its underlying component `Progress` does have that props. 

### What version of @mantine/hooks page do you have in package.json?

5.2.5

### Do you know how to fix the issue

Yes

### Are you willing to participate in fixing this issue and create a pull request with the fix

Yes

### Possible fix

Pass aria-label props to `NavigationProgress` and pass to `Progress`
